### PR TITLE
Anycloud and Alan binary installation

### PR DESCRIPTION
**Pull request by aguillenv**

### CODE REVIEW

File: Formula/alan.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class Alan < Formula
+  desc "Autoscalable Programming Language"
+  homepage "https://github.com/alantech/alan"
+  url "https://github.com/alantech/alan/releases/download/v0.1.34/alan-macos.tar.gz"
+  sha256 "1114cf7ceb5e9d2db92f2a3a38f9e05f33d9efdbb677b367a8e2405530b271e8"
+  license "Apache-2.0"
```
Comment (depombo) [inline Formula/alan.rb]: can we omit the license? it is technically not Apache-2.0

File: Formula/alan.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  revision 1
+
+  def install
+    puts "Installing alan..."
```
Comment (depombo) [inline Formula/alan.rb]: nit `Installing Alan...`

File: Formula/anycloud.rb [hunk 3]
```diff
@@ -1,36 +1,17 @@
...
-    end
+  desc "Elastically scale webservers in any cloud provider"
+  homepage "https://github.com/alantech/anycloud"
+  url "https://github.com/alantech/anycloud/releases/download/v0.1.4/anycloud-macos.tar.gz"
+  sha256 "3167633d004b031d329692d6cbf6cea6c6b4ee0cc13565d8fcd6edf9fe8a4cdd"
+  license "Apache-2.0"
+  revision 1
+
+  def install
+    puts "Installing anycloud..."
```
Comment (depombo) [inline Formula/anycloud.rb]: nit `Installing AnyCloud...`

File: Formula/alan.rb (hunk 1 above)
Comment (dfellis) [reply Formula/alan.rb]: I don't think we can omit. Let's change it to AGPL-3.0

File: Formula/anycloud.rb (hunk 3 above)
Comment (aguillenv) [reply Formula/anycloud.rb]: Done

File: Formula/alan.rb (hunk 2 above)
Comment (aguillenv) [reply Formula/alan.rb]: Done

File: Formula/alan.rb (hunk 1 above)
Comment (aguillenv) [reply Formula/alan.rb]: Set it to AGPL-3.0

**Review by depombo — APPROVED** [review]

**Review by dfellis — APPROVED** [review]

**Pull request merged**